include/load_data.h

```diff
@@ -1,32 +1,30 @@
-
-
 #pragma once
 
-
-#include <string> 
-
-#include <vector> 
-
+#include <string>
+#include <utility>
+#include <vector>
 
 #include <pcl/point_cloud.h>
 
-
-#include <Eigen/dense>
- 
-
-#include "data_types.h"
-
+#if defined __GNUC__ || defined __APPLE__
```

Comment:
Wouldn't it be better to find the Eigen base directory in CMake and add eigen3 to the target_include_path when necessary?
I think it depends on the specific package manager if it ends up in the eigen3 subdir or not, it's not the OS's problem.

Reply:
Indeed, that sounds like a good idea. I only focus on bringing the commits from the other repo here in this PR but this is something we should fix.